composer.json

```diff
@@ -14,7 +14,7 @@
         }
     },
     "require": {
-        "php": "~5.6|~7.0",
+        "php": ">=5.5.0",
```

Comment:
Could you keep the `~5.5|~7.0` notation? That avoids speculating on BC breaks in PHP 8